scripts/kind-build-test.sh

```diff
@@ -15,7 +15,10 @@ source "$SCRIPTS_DIR/lib/k8s.sh"
 
 OPTIND=1
 CLUSTER_NAME_BASE="test"
+AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID:-"123456789012"}
```

Comment:
I feel like we should parse the ROLE_ARN passed by user to get ACCOUNT_ID instead of passing a random string to avoid confusion.